hwdb/60-evdev.hwdb

```diff
@@ -253,6 +253,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# Lenovo ThinkPad T430
```

Comment:
No, tests still fail. Please run `make check` or `parse_hwdb.py` manually to verify.